pkg/quarkus/v1alpha/api_test.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2021 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/spf13/pflag"
+	"sigs.k8s.io/kubebuilder/v3/pkg/config"
+	"sigs.k8s.io/kubebuilder/v3/pkg/machinery"
+	"sigs.k8s.io/kubebuilder/v3/pkg/model/resource"
+)
+
+var _ = Describe("v1", func() {
+	testAPISubcommand := &createAPISubcommand{}
+	testResource := resource.Resource{
+		GVK: resource.GVK{
+			Group:   "test-group",
+			Version: "v1",
+			Kind:    "Test-Kind",
+		},
+		Plural: "test-plural",
+	}
+
+	Describe("UpdateResource", func() {
+		testAPIOptions := &createAPIOptions{
+			CRDVersion: "testVersion",
+			Namespaced: true,
+		}
+		updateTestResource := resource.Resource{}
```

Comment:
I would move these into a `BeforeEach` block. 
```suggestion
    var (
        testAPIOptions     createAPIOptions
        updateTestResource resource.Resource
    )   
    BeforeEach(func() {
        testAPIOptions = &createAPIOptions{
            CRDVersion: "testVersion",
            Namespaced: true,
        }   
        updateTestResource = resource.Resource{}
    })  
```